Formula/ffmbc.rb

```diff
@@ -22,7 +22,6 @@ class Ffmbc < Formula
   depends_on "yasm" => :build
 
   depends_on "x264" => :recommended
-  depends_on "faac" => :recommended
```

Comment:
I didn't check here, but since this ffmbc release is based on a >2yr old fork of FFmpeg, I bet there's faac support.